cloud

```diff
@@ -1796,6 +1800,8 @@ function execute_command() {
 
     declare -r load_queue_trigger_topic="${prefix}load_queue_trigger"
     declare -r cache_bucket_name="${project}_${prefix}cache"
+    declare -r cache_redirect_function_name="${prefix}cache_redirect"
+    declare -r cache_redirector_url="https://${CLOUD_FUNCTION_REGION}-${project}.cloudfunctions.net/${prefix}cache_redirect"
```

Comment:
Please do not duplicate the generation of the function name. Use the `cache_redirect_function_name` variable you define just above. We need to avoid duplication as much as possible so things don't break when, accidentally, only one is updated.

Please also make sure this line doesn't exceed 78 characters in length.